app/src/main/java/com/pokerio/app/utils/GameState.kt

```diff
@@ -415,6 +425,8 @@ object GameState {
         thisPlayer = Player("", "")
         gameCard1 = GameCard.none()
         gameCard2 = GameCard.none()
+        winningsPool = 0
```

Comment:
I think you didn't add this to the `GameStateTest` to see if these values are actually reset

Reply:
Done.